java.base/src/main/java/java/lang/PosixProcessImpl.java

```diff
@@ -119,12 +125,13 @@ LaunchMechanism launchMechanism() {
         }
 
         static Platform get() {
-            String osName = GetPropertyAction.privilegedGetProperty("os.name");
-
-            if (osName.equals("Linux")) {return LINUX;}
-            if (osName.contains("OS X")) {return BSD;}
-            if (osName.equals("AIX")) {return AIX;}
+            if (Build.Target.isLinux()) {
+                return LINUX;
+            } else if (Build.Target.isMacOs()) {
+                return BSD;
+            }
```

Comment:
we set os.name to `darwin` on MacOS (which seems to be expected elsewhere), so the expedient thing was just to rework this code to use the qbicc `Build` macros